plugins/vue-filters.js

```diff
@@ -28,3 +30,25 @@ Vue.filter('optimisedImageUrl', (imageUrl, contentType) => {
   }
   return imageUrl;
 });
+
+Vue.filter('inCurrentLanguage', (val, locale) => {
+  let languageKeys = ['en', 'eng', 'def', 'und'];
+  const currentLocale = locales.find(l => l.code === locale);
+
+  if (currentLocale.code !== 'en' || currentLocale.isoAlpha3 !== 'eng') {
+    languageKeys.unshift(currentLocale.code);
+    languageKeys.unshift(currentLocale.isoAlpha3);
+  }
+
+  for (let key of languageKeys) {
+    if (val[key]) {
+      let htmlLang = '';
+      if (key !== 'def' && key !== 'und') {
+        const langCode = key.length === 3 ? locales.find(l => l.isoAlpha3 === key).code : key;
+        htmlLang = currentLocale.code !== langCode ? langCode : '';
+      }
+      return { 'code': htmlLang, 'value': val[key].join('\n\n').replace(/\n/g, '<br/>') };
```

Comment:
The newline to `<br/>` conversion should only occur on descriptions (for now). I suggest making this a new filter and calling it from the record page view on description only. That will also save us having to use `v-html` all over the place.

Reply:
I have created a new filter.